spec/models/solidus_bolt/payment_source_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::PaymentSource, type: :model do
+  let(:column_list) {
+    [
+      'id',
+      'payment_method_id',
+      'transaction_id',
+      'transaction_reference',
+      'transaction_type',
+      'processor',
+      'date',
+      'transaction_status',
+      'amount',
+      'currency',
+      'created_at',
+      'updated_at'
+    ]
+  }
+
+  describe 'Check Model Integrity' do
+    it 'has correct table name' do
+      expect(described_class.table_name).to eq 'solidus_bolt_payment_sources'
+    end
+
+    it 'has correct attributes' do
+      expect(described_class.new.attributes.keys).to eq column_list
```

Comment:
I've had issues with this test in the past, would prefer to add a `.sort` on both sides

Reply:
added `.sort`